.gitignore

```diff
@@ -27,3 +27,4 @@ yarn-error.log
 yalc.lock
 cypress/screenshots/*
 docker-compose.prod.yml
+docker-compose.yml
```

Comment:
We don't want to ignore `docker-compose.yml` do we? I feel like we should include a safe fail-safe docker-compose config in case someone doesn't know how to run the script to generate one.